asr1k_neutron_l3/models/neutron/l3/interface.py

```diff
@@ -125,8 +125,12 @@ def __init__(self, router_id, router_port, extra_atts):
         super(GatewayInterface, self).__init__(router_id, router_port, extra_atts)
 
         self.nat_address = self._nat_address()
+        
+        # annotate details about the router to the interface description so this can be picked up by SNMP
+        description = (f'type:gw;router:{self.router_id};network:{self.router_port["network_id"]};'
+                       f'subnet:{self._primary_subnet_id}')
```

Comment:
If you want to use fstrings would you be so kind and drop all python versions <3.6 from `setup.cfg` and add 3.6 - 3.10 in an extra commit?

Reply:
I opened PR https://github.com/sapcc/asr1k-neutron-l3/pull/79 for it.